apps/docs/src/guide/wallet-manager/getting-started-with-wallet-manager.md

```diff
@@ -0,0 +1,59 @@
+
+To add a vault, we utilize the `addVault` method. Here's how we can create a Private Key vault and add a private key from a wallet we own:
+
+<<< @/../../docs-snippets/src/guide/wallet-manager/getting-started-with-wallet-manager.test.ts#getting-started-with-wallet-manager-3{ts:line-numbers}
+
+The `addVault` method requires an object with three properties: `type`, `secret`, and `title`. The `WalletManager` currently supports two types of vaults: `privateKeyVault` and `mnemonicVault`. For the `secret`, we use our wallet's private key, and for the `title`, we can provide a custom name.
+
+By running this code, `WalletManager` creates a new vault instance of the type `privateKey` and adds one account (our wallet) to this newly created vault.
+
+A key feature of the `WalletManager` is its ability to manage multiple vaults, even of the same type. This implies that if you run the `addVault` method again, with the same parameters, `WalletManager` will create another vault of the type `privateKey`, holding the same wallet. Here's an example:
+
+<<< @/../../docs-snippets/src/guide/wallet-manager/getting-started-with-wallet-manager.test.ts#getting-started-with-wallet-manager-4{ts:line-numbers}
+
+After executing this, you will find that your `WalletManager` is managing two `privateKey` vaults, both storing the same wallet.
+
+Remember, both `title` and `secret` are optional when adding vaults, but providing a `title` makes it easier to manage your vaults and wallets. If you add a vault without providing a `secret`, this will result in one new account (wallet) being generated by the vault it self.
+
+## Using The `WalletManager`
+
+With your `WalletManager` set up, you can now access your vaults and wallets. Here's how to retrieve the details of your vaults:
+
+<<< @/../../docs-snippets/src/guide/wallet-manager/getting-started-with-wallet-manager.test.ts#getting-started-with-wallet-manager-5{ts:line-numbers}
+
+This will output something like this:
+
+<<< @/../../docs-snippets/src/guide/wallet-manager/getting-started-with-wallet-manager.test.ts#getting-started-with-wallet-manager-6{bash:line-numbers}
+
+As you can see, the WalletManager assigns unique `vaultIds` for each vault. The first vault you added has a `vaultId` of `0`, and the second one has a `vaultId` of `1`.
+
+Let's retrieve your wallet instance with the `getWallet` method:
+
+<<< @/../../docs-snippets/src/guide/wallet-manager/getting-started-with-wallet-manager.test.ts#getting-started-with-wallet-manager-7{ts:line-numbers}
+
+This guide walked through the steps to instantiate a WalletManager, set up its first vault, and retrieve vault information. The following sections will explore more functionalities of WalletManager, examine deeper into the usage of its vaults, and the details of its storage system.
```

Comment:
```suggestion
This guide walked through the steps to instantiate a WalletManager, set up its first vault, and retrieve vault information. The following sections will explore more functionalities of WalletManager, and go deeper into the usage of its vaults and the details of its storage system.
```

Reply:
@arboleya probably also needs ticks for `WalletManager`